R/prepare_pctr_product.R

```diff
@@ -19,7 +19,7 @@ prepare_pctr_product <- function(pctr_prod, comp, eco_activities, match_mapper,
     rename_pctr_product() |>
     mutate(benchmark = ifelse(is.na(.data$PCTR_risk_category), NA, .data$benchmark), .by = c("companies_id")) |>
     select(-c(
-      "matching_certainty_num", "avg_matching_certainty_num", "co2_footprint"
+      "matching_certainty_num", "avg_matching_certainty_num", "co2_footprint", "extra_rowid"
     )) |>
```

Comment:
You may avoid skipping yet another test by being less strict about the presence of `extra_rowid`, e.g.:

```r
    select(-matches("extra_rowid")) |>
    select(-c(
      "matching_certainty_num", "avg_matching_certainty_num", "co2_footprint"
    ))  |>
```

Reply:
@maurolepore We anyway have to refactor this test because this test don't use the new API (here: `profile_emissions`). It uses the old `prepare_pctr_product` function which do not use the `extra_rowid` column. Hence, I would like to refactor this test using this [ticket](https://github.com/2DegreesInvesting/tiltIndicatorBefore/issues/73). Does it sounds good?